<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 'grey'</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exteriorButton1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="pickExteriorLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Pick Exterior Floor:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton8">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exteriorButton10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(130, 180, 255);
font: 24pt &quot;.AppleSystemUIFont&quot;;
border-style: outset;
border-width: 2px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
